templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -16,25 +16,18 @@ PLUGIN_SOURCE="./{{ plugin_name }}/dist/{{ plugin_name | snake }}-${PLUGIN_VERSI
 export PULP_API_ROOT="{{ api_root }}"
 
 PIP_REQUIREMENTS=("{{ cli_package }}")
-if [[ "$TEST" = "docs" || "$TEST" = "publish" ]]
-then
-  PIP_REQUIREMENTS+=("-r" "doc_requirements.txt")
 {%- if plugin_name == 'pulpcore' %}
+if [[ "$TEST" = "publish" ]]
```

Comment:
This sounds ridiculous. There is no "publish" test.

Reply:
It's safe to remove this, then?

Reply:
I believe so. Maybe start with a "git grep publish". But I'm pretty sure this is a leftover from former versions of the release process.